app/Localization/Locale.php

```diff
@@ -7,31 +7,26 @@
 class Locale
 {
     protected $locales = [
-        'en',
-        'es',
-        'pt',
-        'sv',
-        'de',
-        'da',
+        'en' => 'English',
+        'es' => 'Español',
+        'pt' => 'Português',
+        'sv' => 'Svenska',
+        'de' => 'Deutsch',
+        'da' => 'Dansk',
     ];
 
     protected $localeToLanguage = [
-        'en' => "English",
-        'es' => "Spanish",    // [TODO translate]
-        'pt' => "Portuguese", // [TODO translate]
-        'sv' => "Swedish",    // [TODO translate]
-        'de' => "German",    // [TODO translate]
-        'da' => "Danish",    // [TODO translate]
+
     ];
 
     public function all()
     {
-        return $this->locales;
+        return array_keys($this->locales);
```

Comment:
I wonder if this should now be renamed? `keys` or `slugs` instead of `all()`? Sorry, I know I told you to do this but now that I see it I see why it's a bit janky.

Let's make this `slugs` I think :)

Reply:
Done.